spec/controllers/api/v1/map/map_controller_spec.rb

```diff
@@ -6,9 +6,9 @@
     it "returns json" do
       create(:map_state)
       get :index
-      data = JSON.parse(response.body)
+      data = JSON.parse(response.body)["map"]
+
```

Comment:
The format of the map data also changed,  even though no serializer was added for this model.... not sure why?